plugins/modules/lightsail_snapshot.py

```diff
@@ -0,0 +1,207 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: Ansible Project
```

Comment:
```suggestion

# Copyright: Contributors to the Ansible project
```
https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#copyright-and-license

Not a 'must', but since this is a new module it's recommended that the Copyright attribution be "Ansible contributors" rather than the Project.